src/docfx/lib/cache/JsonDiskCache.cs

```diff
@@ -78,14 +78,14 @@ public async Task<(TError? error, TValue? value)> GetOrAdd(TKey key, Func<TKey,
                 return (default, value);
             }
 
-            var error = await Update(key, valueFactory);
+            var error = Update(key, valueFactory).GetAwaiter().GetResult();
```

Comment:
Can we reduce the call of `.GetAwaiter().GetResult()`? like putting this to a higher layer

Reply:
Lifting it outside is gonna increase the # of places that calls `GetAwaiter().GetResult()`, because both `GitHubAccessor` and `MsAliasValidator` calls this method?